clang/lib/3C/ConstraintBuilder.cpp

```diff
@@ -21,14 +21,15 @@ using namespace llvm;
 using namespace clang;
 
 // Used to keep track of in-line struct defs
-unsigned int LastRecordLocation = -1;
+RecordDecl *lastRecord2 = nullptr;
+static std::map<Decl *, Decl *> VDToRDMap2;
+static std::set<Decl *> inlines2;
 
 void processRecordDecl(RecordDecl *Declaration, ProgramInfo &Info,
-                       ASTContext *Context, ConstraintResolver CB,
-                       bool IsInFunction) {
+                       ASTContext *Context, ConstraintResolver CB) {
+  lastRecord2 = Declaration;
```

Comment:
`lastRecord2` is not a great variable name. Also, I think that it should start with a capital letter; can you confirm you are following the Clang/LLVM coding standard? Perhaps check with @mattmccutchen-cci how to run `clang-tidy` or `clang-format` (I forget the names).

Reply:
Right, the standard says that variables should start with a capital letter.  But until we have better automation, I'm not sure it's worth spending the time to catch and fix style-rule issues in individual PRs.  I personally am not bothered by the inconsistencies (though I understand if others on our team are) and they are easy enough for me to fix in bulk before we send an omnibus PR to Microsoft, provided it's obvious what the fix should be.  (The last pass took me a long time because I was learning the process; the next one should be much faster.)  The clang-tidy and clang-format information is [here](https://docs.google.com/document/d/1lnlUSOdKqXV3f8oG0GS0BLyA6peNNCY-kzRpvF8e8e8/edit#heading=h.dosgx1ko0dil) if anyone's curious.

All that said, given that Mike asked to improve this variable name anyway, it's efficient to bring it into style compliance at the same time.